core/example/04_aosoa_openmp/aosoa_openmp.cpp

```diff
@@ -0,0 +1,225 @@
+/****************************************************************************
+ * Copyright (c) 2018 by the Cabana authors                                 *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the Cabana library. Cabana is distributed under a   *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Cabana_Core.hpp>
+
+#include <iostream>
+
+//---------------------------------------------------------------------------//
+// AoSoA example using OpenMP.
```

Comment:
It may be better to make it more explicit by stateing that "AoSoA example using OpenMP parallel for for particle initialization".